src/server/adminBro.ts

```diff
@@ -978,6 +978,7 @@ const sendProjectsToGoogleSheet = async (
       listed: Boolean(project.listed),
       totalDonations: project.totalDonations,
       totalProjectUpdates: project.totalProjectUpdates,
+      website: project.website || '',
```

Comment:
Add to the headers and the params of the addSheetWithRows

Reply:
Thanks for mentioning, Done